Source/SpacePort/SpacePort/Models/Receipt.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace SpacePort.Models
+{
+    public class Receipt
+    {
+        public int ID { get; set; }
+        public int TotalPrice { get; set; }
+        public int Duration { get; set; }
+        public int TicketID { get; set; }
+        public Ticket Ticket { get; set; }
```

Comment:
I believe that the property "TicketID" is not needed since this class already defines its relationship with "Ticket"-class through the "Ticket"-property".

Reply:
We tried, but when we read about one-to-one it showed alot of errors